### DISCUSSION TITLE
delete

### DISCUSSION THREAD

**Discussion by chensijiang** [filecoin-project/lotus#8518] [General Q&A]
delete

**Comment by weizihua** [comment]
How can you prove that the node owner is yours？

**Reply by chensijiang** [reply]
delete

**Reply by chensijiang** [reply]
delete

**Reply by weizihua** [reply]
Track that related FIL assets have flowed into the exchange and can seek help from the exchange to locate an associate.

**Comment by LexLuthr** [comment]
I am archiving this discussion as there has been no new input from the author for more than 7 days.
